src/dune_engine/build_system.mli

```diff
@@ -57,16 +57,14 @@ val run_exn : (unit -> 'a Memo.t) -> 'a Fiber.t
 (** {2 Misc} *)
 
 module Progress : sig
+  (** Measures for the progress of the build. *)
```

Comment:
Can you add a space below this comment so it's attached to the module instead of the type.